contracts/SlashIndicator.sol

```diff
@@ -158,6 +188,73 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function handleFinalityReport(FinalityEvidence calldata _evidence) external onlyInit oncePerBlock noReentrant {
+    require(
+      _evidence.numA + finalityDistance >= _evidence.numB || _evidence.numB + finalityDistance >= _evidence.numA,
```

Comment:
The key problem is that, when we do the fork about this feature, the value of newly defined `finalityDistance`, `finalitySlashRewardRatio` will be default value: zero.  Please handle such situation.